tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
+USERNAME_GRAFANA = os.environ["USERNAME_GRAFANA"]
+PASSWORD_GRAFANA = os.environ["PASSWORD_GRAFANA"]
+SUCCESS_SIGN = "✅"
+ERROR_SIGN = "❌"
+
+def pd_list_users():
+
+    session = APISession(PAGERDUTY_API_TOKEN)
+
+    users = session.list_all("users", params={"include[]": "notification_rules"})
```

Comment:
Do we need the notification rules?
```suggestion
    users = session.list_all("users")
```